lib/modules/dosomething/dosomething_helpers/dosomething_helpers.module

```diff
@@ -697,7 +697,41 @@ function dosomething_helpers_get_current_user() {
     return null;
   }
 
-  return $user;
+  return dosomething_helpers_get_user_info($user->uid);
+}
+
+/**
+ * Get the user information for a specified user from Northstar.
+ *
+ * @param  string $drupal_id
+ * @return null|array
+ */
+function dosomething_helpers_get_user_info($drupal_id) {
+  $northstar_response = dosomething_northstar_get_northstar_user($drupal_id);
+  $northstar_response = json_decode($northstar_response);
+
+  if ($northstar_response && isset($northstar_response->error)) {
+    return NULL;
+  }
+
+  $northstar_user = $northstar_response->data;
+
+  return [
+    'id' => dosomething_helpers_isset($northstar_user, 'id'),
```

Comment:
Definitely not necessary for merge, but might make sense to mark `dosomething_helpers_isset` as deprecated and get out of the habit of using it now that we have the battle-tested tools included in the Collect package.